tools/test.sh

```diff
@@ -21,7 +20,7 @@ function prepare {
   pbjs -t json "webknossos-tracingstore/proto/SkeletonTracing.proto" > "$testBundlePath/SkeletonTracing.proto.json"
   pbjs -t json "webknossos-tracingstore/proto/VolumeTracing.proto" > "$testBundlePath/VolumeTracing.proto.json"
   # --copy-files will copy files that are present in the source dir but are not transpiled (e.g.: json files)
-  BABEL_ENV=test babel "$jsPath" --out-dir "$testBundlePath" --copy-files $additionalParams
+  BABEL_ENV=test babel "$jsPath" --out-dir "$testBundlePath" --copy-files
```

Comment:
suddenly, babel complained about the `--timeout` option which was passed to it. the args to this bash script are passed to `prepare` and ava. I'm not aware of any specific args which `babel` needs, which is why this should work alright.